multinet/multinet/db.py

```diff
@@ -52,12 +71,15 @@ def delete_workspace(name, arango=None):
 @with_client
 def get_workspaces(name, arango=None):
     """Return a list of all workspace names."""
-    sys = db("_system", arango=arango)
-    if name and sys.has_database(name):
-        return [name]
+    if test_db(arango):
+        sys = db("_system", arango=arango)
```

Comment:
Let's change this variable from `sys` to something less ambiguous (so that it doesn't conflict with the Python standard module of the same name).

Reply:
This is what was in the file before and is in lots of other places. Should this be a new issue @waxlamp ?

Reply:
Ah you're right. Yes, let's open an issue to fix that.

Reply:
Opened #127